.github/pull_request_template.md

```diff
@@ -0,0 +1,50 @@
+# Description
+
+Please include a summary of the change and which issue is fixed. 
+
+Fixes # (issue)
+
+## Type of change
```

Comment:
Type of change could be removed here, clustered in 5 classes and used as new templates. So that we could do also do some extra auto-labeling.

Reply:
Oh is that for PR github hasn't an UX like for issue templates: https://github.community/t/multiple-pull-request-templates/1850/5

Reply:
Yeah that would be the preferred method, but we're waiting on GitHub to support that